src/pages/ProductPage/Product.js

```diff
@@ -1,7 +1,289 @@
+            return (
+              <div className="detail-product-outer-cont" key={jelly}>
+                <ProductJelly product={product} />
+              </div>
+            );
+          })}
+        </div>
+      ),
+    },
+
+    {
+      tabTitle: (
+        <ul
+          className={`${'tabs'} ${activeIndex === 0 ? true : ''}`}
+          onClick={() => tabClickHandler(3)}
+        >
+          <li className="product-tab-name">캔디</li>
+        </ul>
+      ),
+      tabCont: (
+        <div>
+          {candy.map((product, candy) => {
+            return (
+              <div className="detail-product-outer-cont" key={candy}>
+                <ProductCandy product={product} />
+              </div>
+            );
+          })}
+        </div>
+      ),
+    },
+
+    {
+      tabTitle: (
+        <ul
+          className={`${'tabs'} ${activeIndex === 0 ? true : ''}`}
+          onClick={() => tabClickHandler(4)}
+        >
+          <li className="product-tab-name">쿠키</li>
+        </ul>
+      ),
+      tabCont: (
+        <div>
+          {cookie.map((product, cookie) => {
+            return (
+              <div className="detail-product-outer-cont" key={cookie}>
+                <ProductCookie product={product} />
+              </div>
+            );
+          })}
+        </div>
+      ),
+    },
+
+    {
+      tabTitle: (
+        <ul
+          className={`${'tabs'} ${activeIndex === 0 ? true : ''}`}
+          onClick={() => tabClickHandler(5)}
+        >
+          <li className="product-tab-name">케이크</li>
+        </ul>
+      ),
+      tabCont: (
+        <div>
+          {cake.map((product, cake) => {
+            return (
+              <div className="detail-product-outer-cont" key={cake}>
+                <ProductCake product={product} />
+              </div>
+            );
+          })}
+        </div>
+      ),
+    },
+  ];
+
+  const tabClickHandler = index => {
+    setActiveIndex(index);
+  };
+
+  const showDropDown = () => {
+    setDropDownMenu(!dropdownMenu);
+  };
+
+  const filterItemIncrease = event => {
+    setFilter(event.target.value);
+    const priceSorting = [...all];
+    const priceCompare = key => (a, b) => {
+      // return a[key] > b[key] ? 1 : a[key] > b[key] ? -1 : 0;
+      return a[key] - b[key];
+    };
+    priceSorting.sort(priceCompare('price'));
+    setAll(priceSorting);
+    // console.log('클릭');
+  };
+
+  const filterItemDecrease = event => {
+    setFilter(event.target.value);
+    const priceSorting = [...all];
+    const priceCompare = key => (a, b) => {
+      // return a[key] > b[key] ? -1 : a[key] < b[key] ? -1 : 0;
+      return b[key] - a[key];
+    };
+    priceSorting.sort(priceCompare('price'));
+    setAll(priceSorting);
+    // console.log('클릭');
+  };
+
+  // fetch
+  return (
+    <section className="product">
+      <div className="product-nav">네브바</div>
+      <div className="product-pic">
+        <img src="./images/cup.jpg" alt="상품메인이미지" />
```

Comment:
- 이미지의 경로는 public 하위의 절대경로로 다음과 같이 나타낼 수 있습니다.
- 이미지의 alt값에는 '이미지', 'img' 등을 포함하시지 않아도 됩니다.
```suggestion
        <img src="/images/cup.jpg" alt="상품" />
```